DOCS.md

```diff
@@ -312,6 +471,34 @@ _**example**_
 
 ```yaml
 # .drone.yml
+
+# Drone 1.0+
+---
+kind: pipeline
+# ...
+steps:
+  - name: deploy-gke
+    image: nytimes/drone-gke
+    environment:
+      # ;)
```

Comment:
I'm curious, what is this?

Reply:
I have no clue, copied it directly from what was written for 0.8. 😄 

Maybe @montmanu might have an idea?